stdlib/int64.gr

```diff
@@ -150,62 +270,128 @@ export let rec mod = (x: Int64, y: Int64) => {
   ret
 }
 
+/**
+ * @section Bits: Functions for operating on the raw bits of Int64s.
+ */
+
+/**
+ * Rotates the value left by the given number of bits.
+ *
+ * @param value: The value to rotate
+ * @param shiftCount: The shiftCount
```

Comment:
This needs a proper description.

```suggestion
 * @param shiftCount: The number of bits to shift
```
Initial description... probably needs wordsmithing by @ospencer

Reply:
Also, why is this called "shiftCount" for rotate functions?